### DISCUSSION TITLE
How to increase the rate limit?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [renovatebot/renovate#20785] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have [many dependency updates](https://github.com/Hongbo-Miao/hongbomiao.com/issues/3259) got rate limited on Renovate Dependency Dashboard:

<img width="650" alt="image" src="https://user-images.githubusercontent.com/3375461/223289402-90e3fcc2-8a08-411f-bee8-ae9ecfc8574b.png">

I have to manually click them to force Renovate to open pull requests.

I was expecting it opens several pull requests every hour. However, currenlty it seems not. You can see the long gap (9 hours) between those two commits. During this time, a lot of pull requests are waiting to be opened and got rate limited. The new pull requests got opened is because I manually checked them in the Dependency Dashboard. 

<img width="1162" alt="image" src="https://user-images.githubusercontent.com/3375461/223289270-720c9d14-73e5-4b52-b9a0-51382240afef.png">

I am wondering if there is a way to increase the rate limit? Thanks! 😃

### What you tried so far.

Manually checked the dependency update pull request in the Dependency Dashboard as workaround way.

I did find [prHourlyLimit](https://docs.renovatebot.com/configuration-options/#prhourlylimit), however, I saw by default it is `0` which means no rate limit.

My renovate.json5 is 

```json5
{
  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
  extends: [
    'config:base',
  ],
  lockFileMaintenance: {
    enabled: true,
    commitMessageExtra: '({{packageFile}})',
  },
  additionalBranchPrefix: '{{parentDir}}-',
  semanticCommitScope: '{{#if parentDir}}{{parentDir}}{{else}}deps{{/if}}',
}
```

### Relevant debug logs

[log.txt](https://github.com/renovatebot/renovate/files/10904239/log.txt)

**Comment by rarkins** [comment, marked as answer]
The preset `config:base` sets `prHourlyLimit` to 2.

In your logs you can see lines like `DEBUG: PR hourly limit remaining: 0` confirming it is the limit being reached

**Reply by hongbo-miao** [reply]
Thanks @rarkins ! 😃
Just curious then why https://docs.renovatebot.com/configuration-options/#prhourlylimit said it is default 0, is it for something else like “config:default”?
Another question, you said “config:base” sets it to 2, but why there is a gap for Renovate to open new pull requests instead of every hour open 2?

**Reply by rarkins** [reply]
It is default 0 if you have no presets or other config defined.

The hosted Renovate app doesn't run every hour on each repo, it averages around every 2-3 hours for scheduled jobs. It does run immediately "on demand", e.g. if you merge any of its PRs

**Reply by hongbo-miao** [reply]
Got it, thanks for clarifying!
